fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/LoadAction.java

```diff
@@ -145,9 +145,6 @@ public Object streamLoad(HttpServletRequest request,
                 }
             }
         }
-        if (needRedirect(request.getScheme())) {
```

Comment:
The other `needRedirecth` functions in this file also need to be processed.

Reply:
done